Source/Core/VideoBackends/Software/Rasterizer.cpp

```diff
@@ -55,10 +55,10 @@ struct SlopeContext
 struct Slope
 {
   Slope() = default;
-  Slope(float f0, float f1, float f2, const SlopeContext& ctx) : f0(f0)
+  Slope(float f0_, float f1_, float f2_, const SlopeContext& ctx) : f0(f0_)
```

Comment:
IMO it’s nicer to make the change to the internal var name instead of the name used in exposed declaration (also what Google coding standard suggests)

Reply:
I can see the argument for this with private members variables, but when they're public this makes the constructor nicer at the expense of making every other access to the struct members uglier.

I believe Google's style guide [agrees](https://google.github.io/styleguide/cppguide.html#Variable_Names) with me here ("Class Data Members" vs "Struct Data Members").